src/lightning_app/cli/cmd_apps.py

```diff
@@ -0,0 +1,81 @@
+import json
+from datetime import datetime
+
+from lightning_cloud.openapi import Externalv1LightningappInstance, V1LightningappInstanceState
+from rich.console import Console
+from rich.table import Table
+from rich.text import Text
+
+from lightning_app.cli.core import Formatable
+from lightning_app.utilities.cloud import _get_project
+from lightning_app.utilities.network import LightningClient
+
+
+class AppManager:
```

Comment:
It's an abstraction I put into place so that I don't put too much logic into the individual CLI command - pretty much like `_run_app`, just as a class.

I'll make it protected :+1:

Reply:
Looks great